.circleci/config.yml

```diff
@@ -78,12 +78,12 @@ commands:
       - store_artifacts:
           path: ./dist
           prefix: ./dist
-  unit-tests:
+  build-checks:
```

Comment:
Should we split these up in a future ticket?

Reply:
I think it's good to combine the build checks (currently include unit/lint/typechecks)
I don't see benefits splitting them up in CircleCI, only downsides - takes more time, duplicates (in this case 3x) all other pre-install routines such as npm install, loading/saving CI cache, etc which takes too much CI minutes.